docs/how-to/environment/use-environments.md

```diff
@@ -0,0 +1,131 @@
+# How to manage your environments using Hatch
+
+-----
+
+[Hatch environments](config/environment/overview.md) are isolated workspaces that can be used for  project tasks including running tests, building documentation and running code formatters and linters.
+
+## Create a default environment  
+When you start using Hatch, you need to create a new 
+default environment. To do this use:
+
+```bash
+hatch env create
+```
+
+Hatch will create a default environment for you to work in. It will also install your package in `-editable` mode in this default environment.  
+
+### The default environment 
+
+Hatch will always use the `default` environment if an environment is not chosen explicitly when running a command. 
+
+For instance, the command below builds your project in the default Hatch environment. 
+
```

Comment:
You made a super important point that hatch will go to default env unless told otherwise.
But I would rather use an example that "proves" this in a negative way.
 i.e., calling a script defined from another env or calling a `pip list` from another env with different dependencies.
 
 Since this is about making a point that is true to ALL usages of the env feature, I think this is a point to make in a tutorial. 
Emphasizing that _how-to_ is for specific purposes to accomplish - like you did with the examples of isolated test env. 

(I know I'm being pedantic ignore me if it's too much - it's Ofek's call anyways)

Reply:
hi there @DahanDv my understand is this is a how-to not a tutorial. as such i think we just want to show people here how to do things rather than a tutorial that might explore various things that work and don't work. i'm definitely open to feedback but based on what @ofek linked to i how to's are more to the point - how to do this thing well.  

i'm of course open to discussion.

Reply:
I moved to this to a brand-new tutorial section since yes I think how-to documents should be more targeted.